appium/tests/specs/mock/setup/RespectsAllowAttesterSearchOnlyForDomains.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('SETUP: ', () => {
         }
       };
       await AppiumHelper.restartApp(processArgs);
-      await MailFolderScreen.clickCreateEmail();
+      await MailFolderScreen.clickCreateEmail(1000);
```

Comment:
@sosnovsky FYI, RespectsAllowAttesterSearchOnlyForDomains test failed in semaphoreci (i noticed this ui test failure 1 time before).
As I checked failure screenshot, MailFolderScreen was taken. That's why `aid-recipients-text-field-to` was not visibile.
To fix this, I increased delayMS.
So